autopilot/autopilot-serverless-inference/autopilot-models-serverless-inference.ipynb

```diff
@@ -0,0 +1,1311 @@
+{
```

Comment:
<p>I would change this to download from the example notebook S3 bucket which we actively maintain - you can download it from s3://sagemaker-sample-files/datasets/tabular/uci_bank_marketing/bank-additional-full.csv - the dataset is the exact same as the one you use here. This is to prevent broken links - we least we are in charge of the sagemaker-sample-files bucket. If anything happens to sagemaker-sample-data-us-west-2, this notebook will be broken</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3677/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
Done. Updated the cell block to use the new s3URI. Instead of `import zipfile` I now use `sagemaker.s3.S3Downloader` to download the dataset from `s3://sagemaker-sample-files/datasets/tabular/uci_bank_marketing/bank-additional-full.csv`